pkg/server/chaosd/redis.go

```diff
@@ -73,9 +108,23 @@ func (redisAttack) Recover(exp core.Experiment, env Environment) error {
 	}
 	attack := config.(*core.RedisCommand)
 
+	cli := redis.NewClient(&redis.Options{
```

Comment:
If the `action` is `core.RedisSentinelStopAction`, this logic is useless. So we should add it to option `core.RedisCacheLimitAction:` logic.

Reply:
Updated!